spec/Section 6 -- Execution.md

```diff
@@ -322,6 +366,49 @@ Unsubscribe(responseStream):
 
 - Cancel {responseStream}
 
+## Yield Subsequent Payloads
+
+If an operation contains subsequent payload records resulting from `@stream` or
+`@defer` directives, the {YieldSubsequentPayloads} algorithm defines how the
+payloads should be processed.
+
+YieldSubsequentPayloads(initialResponse, subsequentPayloads):
+
+- Let {initialRecords} be any items in {subsequentPayloads} with a completed
+  {dataExecution}.
+- Initialize {initialIncremental} to an empty list.
+- For each {record} in {initialRecords}:
+  - Remove {record} from {subsequentPayloads}.
+  - If {isCompletedIterator} on {record} is {true}:
+    - Continue to the next record in {records}.
+  - Let {payload} be the completed result returned by {dataExecution}.
+  - Append {payload} to {initialIncremental}.
+- If {initialIncremental} is not empty:
+  - Add an entry to {initialResponse} named `incremental` containing the value
+    {incremental}.
+- Yield {initialResponse}.
+- While {subsequentPayloads} is not empty:
```

Comment:
Everything under this needs to be indented more?

Reply:
updated